src/animated/Controller.ts

```diff
@@ -69,123 +92,131 @@ class Controller<P extends any = {}> {
           to: { ...previous, ...entry.to },
         }
       })
-      this.queue = Object.values(ops)
```

Comment:
😳 The `update` method should never replace the `queue`. So I converted the `ops` object to a sparse array!